### PR TITLE
syscontainers: Add code documentation

### PR DESCRIPTION
**Pull request by ashcrow**
## Description
Adds docstrings and parameter information to the public methods in syscontainers.py. Added as a way to get more comfortable with the module and hopefully help others get up to speed quicker.

## Related Bugzillas
None

## Related Issue Numbers
None

## Pull Request Checklist:
Not needed

### CODE REVIEW

**Comment by ashcrow** [comment]
cc @giuseppe

File: Atomic/syscontainers.py [hunk 1]
```diff
@@ -925,12 +982,27 @@ def _get_ostree_repo(self):
         return repo
 
     def version(self, image):
+        """
+        Returns information on provided image in ostre. Same as foo = [inspect_system_image(image)].
```
Comment (giuseppe) [inline Atomic/syscontainers.py]: small typo: ostree

File: Atomic/syscontainers.py [hunk 2]
```diff
@@ -152,9 +172,26 @@ def _pull_image_to_ostree(self, repo, image, upgrade):
         return image
 
     def pull_image(self, image=None):
+        """
+        Public method for pulling an image from an external location into ostree.
+
+        :param image: Name of the image to pull. If not provided self.args.image is used.
+        :type image: str or None
+        """
+        # TODO: guard against image and self.args.image being None?
```
Comment (giuseppe) [inline Atomic/syscontainers.py]: apparently tests fail if this line is added

Comment (ashcrow) [reply Atomic/syscontainers.py]: Bah. It's because ``TODO`` is considered bad by the checks. I'll remove it.

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ f188aff

**Pull request closed without merging**